composer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "neos/flow-development-collection",
+    "description": "Flow packages in a joined repository for pull requests.",
+    "type": "neos-package-collection",
+    "require": {
+        "php": ">=5.5.0",
+        "ext-zlib": "*",
+        "ext-SPL": "*",
+        "ext-json": ">=1.2.0",
+        "ext-mbstring": "*",
+        "ext-reflection": "*",
+        "doctrine/orm": "~2.4.0",
+        "doctrine/migrations": "~1.0.0",
+        "symfony/yaml": "~2.5.0",
+        "symfony/dom-crawler": "~2.5.0",
+        "symfony/console": "~2.0",
+        "composer/installers": "~1.0.2"
```

Comment:
Good point, that was a separate change. Ok let me go though that and push everything needed to make this work...